superset/assets/javascripts/explore/components/ExploreViewContainer.jsx

```diff
@@ -24,14 +25,76 @@ const propTypes = {
 };
 
 class ExploreViewContainer extends React.Component {
+
   constructor(props) {
     super(props);
     this.state = {
       height: this.getHeight(),
       showModal: false,
+      table: Array(0),
+      data: [],
     };
   }
 
+  statsPairedTTest(aa, bb) {
```

Comment:
This does not belong in ExploreViewContainer, I think it is specific to `nvd3.jsx` at this point and should probably be defined there. 

This PR shouldn't touch `ExploreViewContainer` at all.